src/components/SponsorLogos.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const SponsorListContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  align-items: center;
+  margin-bottom: 1em;
+`
+
+const SponsorContainer = styled.div`
+  margin: 1em;
+`
+
+const SponsorImage = styled.img`
+  max-width: ${p => p.width}px;
+  transition: transform 0.2s;
+
+  &:hover {
+    transform: scale(1.1);
+  }
+`
+
+const platWidth = 250
```

Comment:
nit: `maxSponsorWidth` might be a better name here

Reply:
agreed